docs/launch_PMM.md

```diff
@@ -0,0 +1,24 @@
+# Launch a Percona Monitoring and Management (PMM) environment
+
+Test-drive PMM from within Percona Platform by generating a free Kubernetes cluster, automatically provisioned with a PMM server, where you can try QAN, DBaaS, Advisor, Alerts and all the features that PMM has to offer.
+
+PMM is an open-source solution for running and managing MySQL and MongoDB clusters on Kubernetes. It provides the observability required to understand database health while offering actionable insights to remediate database incidents or performance issues.  
```

Comment:
The first sentence looks like we mixing PMM with DBaaS 'PMM is an open-source solution for running and managing MySQL and MongoDB clusters on Kubernetes.'